chaos-monkey-docs/src/main/asciidoc/changes.adoc

```diff
@@ -10,4 +10,6 @@
 === Contributors
 This release was only possible because of these great humans:
 
+https://github.com/Abijithkrishna
```

Comment:
Please use a markdown link here (also, this should be a list) like
`- https://github.com/Abijithkrishna[@Abijithkrishna]`.